web-console/4.0-designs/storage/snapshots/snapshot.md

```diff
@@ -0,0 +1,105 @@
+# PVC/PV Snapshots
+ 
+## Main use cases
+Create snapshot (from where PVC is)
+List / View related snapshot(s) for a given PVC
+Delete snapshot(s)
+Create new PVC from a snapshot (import/restore snapshot)
+ 
+## User Stories
+As a Developer, I want to be able to create a snapshot (from where PVC is) to make a point-in-time copy of my data.  
+As a Developer, I want to create a new PVC from a snapshot (import/restore snapshot)
+As a Developer, I want to delete 1 or more snapshot(s) when I no longer need to use the snapshot(s).
+As a Developer, I want to be able to list and view my snapshots for a given PVC.
+As a Developer, I want to modify the schedule of a previously-scheduled snapshot.
+As a Developer, I want to know how much capacity the snapshots are taking. [Dashboard support for snapshots]
+As a Developer, I want to see recent snapshot-related events/activities. 
+
+## PVC/PV Snapshot Workflow 
+### List Pages 
+![PVC List Page](img/PVC-List-01.png)
```

Comment:
@yuvalgalanti this is a small adjustment, but I've seen the new PF blue being used in other parts of the flow, so it would be best to be aligned with the other pages for consistency. Here is the blue: https://www.patternfly.org/v4/design-guidelines/styles/colors

and here are the button examples once they are updated: https://www.patternfly.org/v4/documentation/react/components/button/